### PR TITLE
version 1.2.8 for PhpStorm 2023.1

### PR DESCRIPTION
**Pull request by Danil42Russia**

### CODE REVIEW

**Comment by unserialize** [comment]
> for PhpStorm 2032.1
Just a near future 😀

File: src/test/fixtures/strict_typing/return_statement_4.fixture.php [hunk 1]
```diff
@@ -33,15 +33,14 @@ class MorePointEx extends MorePoint {
 
 }
 
-// this seems failing since 2021.1, it's a PhpStorm bug
-/** @return MorePointEx */
+/** @return Point */
```
Comment (unserialize) [inline src/test/fixtures/strict_typing/return_statement_4.fixture.php:36]: As I remember, the purpose of this test was to return a derived class, not a base class

**Review by unserialize — APPROVED** [review]

File: CHANGELOG.md [hunk 2]
```diff
@@ -2,6 +2,10 @@
 
 ## [Unreleased]
 
+## [1.2.8] - 2022-07-01
```
Comment (Danil42Russia) [inline CHANGELOG.md:5]: oops.. wrong month :)

**Review by unserialize — APPROVED** [review]

File: CHANGELOG.md [hunk 3]
```diff
@@ -2,6 +2,10 @@
 
 ## [Unreleased]
 
+## [1.2.8] - 2022-08-01
```
Comment (unserialize) [inline CHANGELOG.md:5]: wrong year

Comment (Danil42Russia) [reply CHANGELOG.md]: fixed

**Review by unserialize — APPROVED** [review]

**Pull request merged**